packages/aws-cdk-lib/aws-codebuild/lib/project.ts

```diff
@@ -2033,6 +2033,16 @@ export class WindowsBuildImage implements IBuildImage {
     imageType: WindowsImageType.SERVER_2019,
   });
 
+  /**
+   * The standard CodeBuild image `aws/codebuild/windows-base:2019-3.0`, which is
+   * based off Windows Server Core 2019.
+   */
+  public static readonly WIN_SERVER_CORE_2019_BASE_3_0: IBuildImage = new WindowsBuildImage({
+    imageId: 'aws/codebuild/windows-base:2019-3.0',
+    imagePullPrincipalType: ImagePullPrincipalType.CODEBUILD,
+    imageType: WindowsImageType.SERVER_2019,
+  });
+
```

Comment:
Can you please also add support for `aws/codebuild/windows-base:2022-1.0`?

Reply:
`aws/codebuild/windows-base:2022-1.0` requires an alternate configuration -- this requires reserved capacity. This should be added as a dedicated change.

I don't have time to validate that adding `aws/codebuild/windows-base:2022-1.0` works correctly, so I'm not inclined to add it to this change.